src/main/java/org/gaul/s3proxy/junit/S3ProxyRule.java

```diff
@@ -16,26 +16,15 @@
 
 package org.gaul.s3proxy.junit;
 
-import java.io.File;
-import java.io.IOException;
 import java.net.URI;
-import java.nio.file.Files;
-import java.util.Properties;
 
-import com.google.common.annotations.Beta;
-
-import org.apache.commons.io.FileUtils;
-import org.eclipse.jetty.util.component.AbstractLifeCycle;
 import org.gaul.s3proxy.AuthenticationType;
-import org.gaul.s3proxy.S3Proxy;
-import org.jclouds.ContextBuilder;
-import org.jclouds.blobstore.BlobStore;
-import org.jclouds.blobstore.BlobStoreContext;
-import org.jclouds.blobstore.domain.StorageMetadata;
 import org.junit.rules.ExternalResource;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.google.common.annotations.Beta;
```

Comment:
@gaul so we still want the "beta" annotation on the rule? If so (or even if not) should we apply it to the extension?